public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -308,12 +309,8 @@ const Policy = (props: PolicyComponentProps) => {
                     )}
                     {dropdownMenuActions.length > 0 && (
                       <Dropdown overlay={<Menu>{dropdownMenuActions}</Menu>}>
-                        <Button
-                          icon="ellipsis-h"
-                          variant="secondary"
-                          size="sm"
-                          type="button"
-                          aria-label="more-actions"
+                        <MoreButton
```

Comment:
Made this consistent with other parts of the UI. Not using the ellipsis icon anymore for here